src/evict/evict_page.c

```diff
@@ -695,7 +672,7 @@ __evict_review(WT_SESSION_IMPL *session, WT_REF *ref, uint32_t evict_flags, bool
      * Success: assert that the page is clean or reconciliation was configured to save updates.
      */
     WT_ASSERT(
-      session, !__wt_page_is_modified(page) || LF_ISSET(WT_REC_LOOKASIDE | WT_REC_UPDATE_RESTORE));
+      session, !__wt_page_is_modified(page) || LF_ISSET(WT_REC_LOOKASIDE | WT_REC_IN_MEMORY));
```

Comment:
I don't understand this change, either.

Reply:
I want to keep the same behaviour for in memory database as it is useless to do lookaside for in memory database. I removed the WT_REC_UPDATE_RESTORE flag so I'm using WT_REC_IN_MEMORY for the same purpose here.